### PR TITLE
Feature: Add connect server

### PR DESCRIPTION
**Pull request by gary-lloyd**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var http = require('http');
+var url = require('url');
+
+http.createServer(function (req, res) {
+	console.log('Request URL: ' + req.url);
+	var headers = {
+		'Location': '/'
+	};
+
+	var query = url.parse(req.url,true).query;
```
Comment (fosj) [inline src/server.js]: missing space between `req.url,true`

File: src/server.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	
+	if (query.token) {
+		headers['Set-Cookie'] = 'JSESSIONID=' + query.token + '; Path=/; HttpOnly';
+	}
+
+	console.log('Response Headers: ');
+	console.log(headers);
+
+	if (query.noredirect) {
```
Comment (fosj) [inline src/server.js:19]: Indentation looks wrong.

**Review by fosj — CHANGES_REQUESTED** [review]
Do you want to commit the lock file?

**Comment by gary-lloyd** [comment]
Changes made

**Review by fosj — APPROVED** [review]

**Pull request merged**